### PR TITLE
Update help text in commands

### PR DESCRIPTION
**Pull request by azinneera**
## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java [hunk 1]
```diff
@@ -86,8 +86,8 @@ public TestCommand(Path projectPath, PrintStream outStream, PrintStream errStrea
             "dependencies.")
     private boolean offline;
 
-    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve dependencies.")
-    private boolean skipLock;
+//    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve dependencies.")
```
Comment (praneesha) [inline cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java:89]:
```suggestion
//    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve the dependencies.")
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 2]
```diff
@@ -15,21 +14,18 @@ DESCRIPTION
        If a template is not specified, this command creates a module
        with a 'main' function.
 
-       There are two built-in templates named: 'main' and 'service'.
+       There are three built-in templates named: 'main', 'service' and 'lib''.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help:17]:
```suggestion
       There are three built-in templates named: 'main', 'service', and 'lib''.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 3]
```diff
@@ -15,21 +14,18 @@ DESCRIPTION
...
 
        You can download templates from Ballerina Central as well.
        In that case <template-name> becomes a qualified module name.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help:20]:
```suggestion
       In that case, <template-name> becomes a qualified module name.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help [hunk 4]
```diff
@@ -15,21 +14,18 @@ DESCRIPTION
...
 
-           <org-name>/<module-name>[:<version>]
+           <org-name>/<package-name>[:<version>]
 
 
 OPTIONS
        -t <template-name>, --template <template-name>
            Add a module using a predefined template.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help:27]:
```suggestion
           Add a module using a pre-defined template.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 5]
```diff
@@ -29,26 +28,15 @@ OPTIONS
            Write output to the given file. The provided output
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:28]:
```suggestion
           Write the output to the given file. The provided output
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 6]
```diff
@@ -29,26 +28,15 @@ OPTIONS
...
            filename may or may not contain the ‘.jar’ extension.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:29]:
```suggestion
           file name may or may not contain the ‘.jar’ extension.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 7]
```diff
@@ -29,26 +28,15 @@ OPTIONS
...
-           path.
-
        -c, --compile
            Compile the source without generating executable(s).
 
-       -a, --all
-           Build or compile all the modules of a project.
-
        --offline
            Build offline without downloading dependencies.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:35]:
```suggestion
           Build offline without downloading the dependencies.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 8]
```diff
@@ -81,20 +69,18 @@ EXAMPLES
        Compile a Ballerina source file and create an executable jar.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:69]:
```suggestion
       Compile a Ballerina source file and create an executable JAR.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 9]
```diff
@@ -81,20 +69,18 @@ EXAMPLES
...
           $ ballerina build hello.bal
 
-       Build the 'hello' module. This will generate a hello.jar file
-       in the '<project-root>/target/bin' directory.
-          $ ballerina build hello
+       Build the 'hello' package that has an entry point. This will generate a
+        hello.jar file in the '<project-root>/target/bin' directory.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help:73]:
```suggestion
        `hello.jar` file in the '<project-root>/target/bin' directory.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 10]
```diff
@@ -81,10 +77,10 @@ EXAMPLES
        Run the 'main' function and/or services in the hello executable jar.
           $ ballerina run hello.jar
 
-       Run the 'main' function and/or services in the hello module.
-          $ ballerina run hello
+       Run the 'main' function and/or services in the current packge.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:80]:
```suggestion
       Run the 'main' function and/or services in the current package.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 11]
```diff
@@ -81,10 +77,10 @@ EXAMPLES
...
+          $ ballerina run .
 
-       Run the 'main' function in the math module with three program args:
+       Run the 'main' function in the math package with three program args:
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:83]:
```suggestion
       Run the 'main' function in the `math` package with three program args:
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help [hunk 12]
```diff
@@ -81,10 +77,10 @@ EXAMPLES
...
        add, 10 and 5. If the second and third parameters of the 'main'
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help:84]:
```suggestion
       add, 10, and 5. If the second and third parameters of the 'main'
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 13]
```diff
@@ -2,22 +2,19 @@ NAME
...
 SYNOPSIS
-       ballerina test [<options>] <module-name> [<config properties>]
-       ballerina test -a | --all [<options>] [--] [<config properties>]
-       ballerina test <balfile> [<config properties>]
+       ballerina test [<options>]
+                      [<ballerina-file-path> | <ballerina-project-directory>] [<config properties>]
 
 
 DESCRIPTION
        Test compiles and executes Ballerina test sources and print a summary of
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:10]:
```suggestion
       Test compiles and executes Ballerina test sources and prints a summary of
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 14]
```diff
@@ -2,22 +2,19 @@ NAME
...
-       test results. You can execute tests in a module,
-       or an entire Ballerina project.
+       test results. You can execute tests in a Ballerina package or a standalone,
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:11]:
```suggestion
       test results. You can execute tests in a Ballerina package or a standalone
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 15]
```diff
@@ -2,22 +2,19 @@ NAME
...
+       Ballerina file.
     
-       Use 'ballerina test <module-name>' to execute tests in the
-       specified module, or '-a' or '-all' to execute tests in all modules.
+       Use 'ballerina test to execute tests in the current package.
 
        Use 'ballerina test <balfile>' to execute the given Ballerina test file.
               Note that testing single bal files will not work within a Ballerina project.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:17]:
```suggestion
              Note that testing single BAL files will not work within a Ballerina project.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 16]
```diff
@@ -60,11 +57,6 @@ OPTIONS
             Executes only the previously failed test cases for the specified
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:57]:
```suggestion
            Executes only the previously-failed test cases for the specified
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 17]
```diff
@@ -97,30 +86,27 @@ CONFIG PROPERTIES
...
+           $ ballerina test
 
-        List all test groups in math module.
-           $ ballerina test --list-groups math
+        List all test groups in the package.
+           $ ballerina test --list-groups
 
-        Run only the set of tests belonging to group_x in current project.
-           $ ballerina test --groups group_x  --all
+        Run only the set of tests belonging to group_x in current package.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:95]:
```suggestion
        Run only the set of tests belonging to the `group_x` in the current package.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 18]
```diff
@@ -97,30 +86,27 @@ CONFIG PROPERTIES
...
+           $ ballerina test --groups group_x
 
-        Run only the given set of functions in current project.
-            $ ballerina test --tests testFunction --all
+        Run only the given set of functions in current package.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:98]:
```suggestion
        Run only the given set of functions in the current package.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 19]
```diff
@@ -97,30 +86,27 @@ CONFIG PROPERTIES
...
+            $ ballerina test --tests testFunction
 
-        Run only the previously failed test cases in math module.
-            $ ballerina test --rerun-failed math
+        Run only the previously failed test cases in the current package.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:101]:
```suggestion
        Run only the previously-failed test cases in the current package.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 20]
```diff
@@ -97,30 +86,27 @@ CONFIG PROPERTIES
...
+            $ ballerina test --rerun-failed
 
         Run tests in single bal file.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:104]:
```suggestion
        Run tests in a single BAL file.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help [hunk 21]
```diff
@@ -97,30 +86,27 @@ CONFIG PROPERTIES
...
            $ ballerina test main_test.bal
 
         Run tests and generate a test report in addition.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help:107]:
```suggestion
        Run tests and generate a test report.
```

File: cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java (hunk 1 above)
Comment (azinneera) [reply cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java:89]: This is a commented code block, hence updating is not required.

**Review by pramodya1994 — APPROVED** [review]

**Pull request merged**